src/Makefile.am

```diff
@@ -30,6 +34,12 @@ EXTRA_DIST = mp.pl
 mp.S	: mp.pl
 	./mp.pl mp.S
 
+if ICA_FIPS
+all-local: $(LTLIBRARIES) 
```

Comment:
Shouldn't this be something like
```
.libs/.libica.so.$(VERSION1).hmac: $(LTLIBRARIES):
	$(AM_V_GEN) openssl dgst -sha256 ...

.libs/.libica.so.$(MAJOR).hmac: .libs/.libica.so.$(VERSION1).hmac
	cd .libs && ln -sf .libica.so.$(VERSION1).hmac .libica.so.$(MAJOR).hmac

all-local: .libs/libica.so.$(VERSION1).hmac .libs/.libica.so.$(MAJOR).hmac
```

Reply:
Following up our internal discussion I'm pushing some changes. Coding the two hmac file targets like above causes build errors, so I'm just using:
```
hmac-file-lnk: hmac-file
	$(AM_V_GEN) cd ${top_builddir}/src/.libs && ln -sf .libica.so.$(VERSION1).hmac .libica.so.$(MAJOR).hmac

hmac-file: libica.la
	$(AM_V_GEN) openssl dgst -sha256 -mac hmac -macopt hexkey:00000000 ${top_builddir}/src/.libs/libica.so.$(VERSION1) | sed -e 's/^.* //' > ${top_builddir}/src/.libs/.libica.so.$(VERSION1).hmac

hmac_files = hmac-file hmac-file-lnk

all-local: $(hmac_files)  
```

Reply:
But then `hmac-file-lnk` and `hmac-file` should be PHONY-targets since they do not actually create files with exactly that name.

Reply:
OK, added .PHONY for the two targets